src/main/resources/db/changelog/changeset/01-init-problem.yaml

```diff
@@ -4,13 +4,14 @@ databaseChangeLog:
       author: Denis
       changes:
         - createTable:
-            tableName: question
+            tableName: question_entity
```

Comment:
Minor: Is this naming conversion decided in team? Which benefits adds this _entity suffix?

Reply:
Doesn't provide any benefits. Seemed more aesthetic

Reply:
Decided to use the old name